src/cmd/services/m3coordinator/downsample/downsampler.go

```diff
@@ -63,7 +63,8 @@ func NewDownsampler(
 
 func (d *downsampler) NewMetricsAppender() MetricsAppender {
 	return newMetricsAppender(metricsAppenderOptions{
-		agg:                     d.agg.aggregator,
+		agg: d.agg.aggregator,
```

Comment:
weird that this didn't align, did `go fmt` break on this?

Reply:
Honestly, no idea.  `gofmt` was the tool that aligned it like this.  Let's just leave it for now since that how it wants to align it?

Reply:
yep fair enough